tests/EventTest.php

```diff
@@ -253,6 +254,18 @@ public function gettersAndSettersDataProvider(): array
         ];
     }
 
+    public function testBacktraceSetter(): void
```

Comment:
As a convention we are used to name the test methods `test[name of the method]` or `test[name of the method][what is testing]`. Can you kindly change it to `testSetStacktrace`?

Reply:
fixed.